features/machine/machine.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Machine features testing
+
+  # @author jhou@redhat.com
+  # @case_id OCP-21196
+  @smoke
+  Scenario: Machines should be linked to nodes
+    Given I have an IPI deployment
+    Then the machines should be linked to nodes
+
+  # @author jhou@redhat.com
+  # @case_id OCP-22115
+  @smoke
+  Scenario: machine-api clusteroperator should be in Available state
+    Given evaluation of `cluster_operator('machine-api').condition(type: 'Available')` is stored in the :co_available clipboard
+    Then the expression should be true> cb.co_available["status"]=="True"
+
+    Given evaluation of `cluster_operator('machine-api').condition(type: 'Degraded')` is stored in the :co_degraded clipboard
+    Then the expression should be true> cb.co_degraded["status"]=="False"
+
+    Given evaluation of `cluster_operator('machine-api').condition(type: 'Upgradeable')` is stored in the :co_upgradable clipboard
+    Then the expression should be true> cb.co_upgradable["status"]=="True"
+
+    Given evaluation of `cluster_operator('machine-api').condition(type: 'Progressing')` is stored in the :co_progressing clipboard
+    Then the expression should be true> cb.co_progressing["status"]=="False"
+
+  # @author jhou@redhat.com
+  # @case_id OCP-25436
+  @admin
+  @destructive
+  Scenario: Scale up and scale down a machineSet
+    Given I have an IPI deployment
+    And I store all machinesets to the :machinesets clipboard
+
+    Given evaluation of `cb.machinesets[0]` is stored in the :machineset clipboard
```

Comment:
you don't need to store first machine set to clipboard. You can just call `machine_set` whenever you need it. e.g. in the next line you can do
```
And evaluation of `machine_set.desired_replicas` is stored in the :replicas_original clipboard
```